index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="pt-BR">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./style.css">
+  <title>Project Lessons Learned</title>
+</head>
+
+<body>
+
+  <header id="cabecalho">
+    <h1 id="titulo">Projeto - Lessons Learned</h1>
+    <nav>
+    <a href="#home" class="menu">Home</a>
+    <a href="#artigos" class="menu">Artigos</a>
+    </nav>
+  </header>
+
+  <main class="perfil" id="home">
+
+    <div class="img-wrapper">
+      <img src="./perfil.jpeg" alt="Foto de Patrick" id="minha_foto">
+      <p>Patrick Morais Nunes</p>
+    </div>
+    <div class="licoes">
+
+      <section id="licoes_aprendidass">
```

Comment:
Você poderia usar um id como esse:
```suggestion
      <section id="section_licoes_aprendidas">
```
Mas também existem outras formas interessantes de se nomear. Poderia fazer o uso de classes para reaproveitar códigos e evitar conflitos:
```suggestion
      <section class="licoes_aprendidas">
```

Reply:
Valeu pelas dicas!! É meio foi uma gambiarra nos nomes hahahah mas vou melhorar. Obrigado pelo review!